components/anchor/anchor-link.component.ts

```diff
@@ -26,6 +27,8 @@ import { NzAnchorComponent } from './anchor.component';
   selector: 'nz-link',
   exportAs: 'nzLink',
   preserveWhitespaces: false,
+  standalone: true,
+  imports: [NgIf, PlatformModule],
```

Comment:
I discovered that PlatformModule is actually an empty module, let's delete it safely :)